src/styles/pages/pdp.scss

```diff
@@ -0,0 +1,19 @@
+// Product
```

Comment:
On the first load of [the PDP](https://preview-480--base.preview.vtex.app/apple-magic-mouse-99988212/p) we're missing the badge and shelf styles:
<img src="https://user-images.githubusercontent.com/381395/163295044-137bb9b6-9cdb-4ad0-8c00-483db11bdfb0.png" width="50%">

Reply:
I think `@import "src/components/product/ProductCard/product-card.scss";` is missing

Reply:
Tested now and it's been resolved!